controllers/storagecluster/cephcluster_test.go

```diff
@@ -1041,29 +1041,40 @@ func TestParsePrometheusRules(t *testing.T) {
 }
 
 func TestChangePrometheusExprFunc(t *testing.T) {
```

Comment:
Can we change the test function name as well, something like `TestChangePrometheusRulesFunc` ?